Intersect (Core)/GameObjects/SpellBase.cs

```diff
@@ -59,6 +60,16 @@ public AnimationBase HitAnimation
             set => HitAnimationId = value?.Id ?? Guid.Empty;
         }
 
+        [Column("OverTimeAnimation")]
+        public Guid OverTimeAnimationId { get; set; }
+
+        [JsonIgnore]
```

Comment:
Needs to be both `JsonIgnore` and `NotMapped`

Reply:
Mhmm, done